tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es2015",
+    "module": "es2015",
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": false,
+    "moduleResolution": "node",
+    "stripInternal": true,
+    "preserveConstEnums": true,
+    "listFiles": true,
+    "declaration": true,
+    "removeComments": true,
+    "lib": ["es2015", "dom"]
```

Comment:
@niieani, I can't find lib from [official compiler options list](https://www.typescriptlang.org/docs/handbook/compiler-options.html) - does it have any effect? Where can I find more information about it?